examples/kafka_plugin/clustered/main.go

```diff
@@ -115,33 +117,45 @@ type Deps struct {
 // Init is the entry point into the plugin that is called by Agent Core when the Agent is coming up.
 // The Go native plugin mechanism that was introduced in Go 1.8
 func (plugin *ExamplePlugin) Init() (err error) {
-	topic := "example-clustered-topic"
+	topicSync := "example-sync-clustered-topic"
+	topicAsync := "example-async-clustered-topic"
 	// Init channels required for async handler
-	plugin.asyncMessageChannel = make(chan messaging.ProtoMessage, 0)
+	plugin.asyncSuccessChannel = make(chan messaging.ProtoMessage, 0)
 	plugin.asyncErrorChannel = make(chan messaging.ProtoMessageErr, 0)
 
 	connection := plugin.Kafka.NewProtoConnection("example-proto-connection")
 
 	// Create a synchronous publisher for the selected topic and partition
-	plugin.kafkaSyncPublisher = connection.NewSyncPublisherToPartition(topic, 0)
+	plugin.kafkaSyncPublisher = connection.NewSyncPublisherToPartition(topicSync, 0)
```

Comment:
I am thinking, topicSync might be confusing (sync sync is allready used mainly in resync and/or datasync).